### PR TITLE
Fix bug in NNICTLDOC and add several examples

### PR DESCRIPTION
**Pull request by PurityFan**
Fix issue #953

### CODE REVIEW

**Comment by SparkSnail** [comment]
Looks good to me, thanks for the update!

File: docs/en_US/NNICTLDOC.md [hunk 1]
```diff
@@ -47,13 +48,34 @@ nnictl support commands:
...
+    > create a new experiment with the default port: 8080
+
+    ```bash
+    nnictl create --config nni/examples/trials/mnist/config.yml
+    ```
+
+    > create a new experiment with specified port of restful server
+
+    ```bash
+    nnictl create --config nni/examples/trials/mnist/config.yml --port $8088
```
Comment (chicm-ms) [inline docs/en_US/NNICTLDOC.md]: why '$8088' instead of '8088' ?

Comment (PurityFan) [reply docs/en_US/NNICTLDOC.md]: In a regular expression "$" means replaceable, let the user know that this value is replaceable.

Comment (chicm-ms) [reply docs/en_US/NNICTLDOC.md]: '8088' is a literal value, it is weird to be used as a variable,  either use plain literal value '8088' ,  or use [PORT] here to imply it is a replacable variable to be consistent with other parts.

Comment (PurityFan) [reply docs/en_US/NNICTLDOC.md]: Remove "$", now use 8088 as an example, and use [experiment_id] to refer a replaceable value.

**Pull request merged**